telemetry/specs/telemetry_report.mdk

```diff
@@ -510,6 +526,132 @@ particular deployment.
   : The device local time when the packet was first received on the ingress
     physical or logical port, in nanoseconds.
 
+## Telemetry Report Header (Ver 2.0) (16+ octets)
+
+
+       0                   1                   2                   3
+       0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+      +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+      | Ver=2 |Length |NProt| RepMdBits | Reserved|L|D|Q|F|   hw_id   |
+      +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+      |                           Switch id                           |
+      +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+      |                        Sequence Number                        |
+      +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+      |                       Ingress Timestamp                       |
+      +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+      |                  Variable Optional Metadata                   |
+      +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+      | Nprot |   Extension Length    |      Domain Specific ID       |
+      +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+      |   Domain Specific Md  Bits    |   Domain Specific Md Status   |   
+      +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+      |                        Extension Data                         |
+      +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+      |                        Extension Data                         |
+      +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+      |                        Extension Data                         |
+      +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+      
+
+* Ver: Version
+  : This specification defines **version 2**.
+
+* Length
+  : Indicates the length of the telemetry report header in multiples of
+    4 octets, including the variable optional metadata.
+
+* NProt: Next Protocol
+  :
+    * 0: Ethernet
+    * 1: IPv4
+    * 2: IPv6
+    * 3: Telemetrty Extensions
+    * 4-14: Reserved for future use
+    * 15: None
+
+* RepMdBits: Report Metadata Bits
+  : Bitmap that indicates which optional metadata is present in the
+    telemetry report header. Each bit represents 4 octets of optional
+    metadata.
```

Comment:
except for bit 7 which represents 8 octets of optional metadata.

Reply:
Added